**Feature:** Add `maxOptions` props to Select component (#904)
contiamo/operational-ui: src/Select/README.md

;<ComponentWithSelect />
```

> ### With maxOptions param

Reviewer: ```suggestion
### With `maxOptions` param
```

> If you have a huge list from the backend, you can limit the number of options displayed to avoid rendering performance issues.

Reviewer: ```suggestion
If you have a huge list from a backend, you can limit the number of options displayed to avoid rendering performance issues.
```

> Just add `maxOptions=`, and it's done. Please note that without the `filterable` options enable, some options can't be select.

Reviewer: ```suggestion
Just add `maxOptions=` and it's done. Please note that without the `filterable` option enabled, some options can't be selected.
```
